support/builder/config.sh

```diff
@@ -326,7 +326,7 @@ EOT
 
 mkdir -p /hab/svc/builder-worker
 cat <<EOT > /hab/svc/builder-worker/user.toml
-auth_token = "$WORKER_AUTH_TOKEN"
+auth_token = "$GITHUB_CLIENT_SECRET"
```

Comment:
These are actually different things. The `WORKER_AUTH_TOKEN` is a personal access token from GitHub. We'll eventually have the worker to auth through use of a shared secret key that we config the font-end with, but for now :(